src/test/java/org/wltea/analyzer/core/CharacterUtilTest.java

```diff
@@ -0,0 +1,44 @@
+package org.wltea.analyzer.core;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+public class CharacterUtilTest {
+
+  @Test
+  public void testIdentifyCharType1() {
+    Assert.assertEquals(1, CharacterUtil.identifyCharType('1'));
```

Comment:
To get to the final return statement of the method in line 83 `return CHAR_USELESS;` use this `Assert.assertEquals(0, CharacterUtil.identifyCharType(' '));`